pom.xml

```diff
@@ -198,6 +197,12 @@
                 <artifactId>json</artifactId>
                 <version>20090211</version>
             </dependency>
```

Comment:
Note: you would need to change the dependency version of gateway in enterprise.gateway to "develop-SNAPSHOT".  

But I understand how it would work now, so I'll merge it.  Thanks